sigma/pipelines/netwitness/__init__.py

```diff
@@ -1,7 +1,9 @@
 """Mappings of identifier to pipelines used by pySigma"""
 
-from .windows import netwitness_windows_pipeline
+from .fortinet import netwitness_fortinet_pipeline  # pylint:disable=import-error
+from .windows import netwitness_windows_pipeline  # pylint:disable=import-error
```

Comment:
Why do you disable the import errors here?

Reply:
Because pySigma forces you to specify the imports like that and pylint complains that this is not ok.. so I disabled the errors.